deploy/helm/values.yaml

```diff
@@ -408,3 +408,20 @@ swoagent:
     requests:
       memory: 800Mi
       cpu: 100m
+
+fargate:
```

Comment:
i think we should call this "aws_fargate" to avoid any confusion with non-fargate k8s users & provide better clarity.

Reply:
renamed